plugins/woocommerce-admin/client/payments-welcome/index.tsx

```diff
@@ -145,9 +148,19 @@ const ConnectPageOnboarding = ( {
 			wpcom_connection: isJetpackConnected ? 'Yes' : 'No',
 		} );
 
+		recordEvent( 'wcpay_extension_installed', {
```

Comment:
Do you think we should record this event after we actually successfully install the extensions (For example in the `if ( installAndActivateResponse?.success )` block of code)?

Reply:
Good point 👍

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/34581/commits/eabb148f3b86602d3ec84ecf16735b79d86a0382.